server/middleware.json

```diff
@@ -0,0 +1,30 @@
+{
+  "initial:before": {
+    "loopback#favicon": {},
+    "loopback#compress": {}
```

Comment:
Most of these `loopback#xxxxx` shorcuts are deprecated, use the middleware module name instead please. See [docs](https://docs.strongloop.com/display/LB/Defining+middleware#Definingmiddleware-Usingbuilt-inmiddleware) for more details.

Note that `loopback#favicon` is a special case, it provides a default favicon and thus it should be loaded via `loopback#favicon`.